dao/src/main/kotlin/tables/runs/scanner/ScannerConfigurationsTable.kt

```diff
@@ -33,7 +33,7 @@ import org.jetbrains.exposed.dao.id.LongIdTable
 object ScannerConfigurationsTable : LongIdTable("scanner_configurations") {
     val scannerRunId = reference("scanner_run_id", ScannerRunsTable)
 
-    val skipConcluded = bool("skip_concluded").default(false)
+    val skipConcluded = bool("skip_concluded")
```

Comment:
Commit message:
- Typo in "unnnecessary".
- Maybe say "Remove unnecessary `default` *calls*"